### DISCUSSION TITLE
求大佬给个配队和抽取建议(Q-Q)

### DISCUSSION THREAD

**Discussion by renwang123** [share121/inter-knot#2397] [其他]
![11](https://github.com/user-attachments/assets/fccec0c4-ae26-4abf-b3bd-6f3573a400ab)

**Comment by renwang123** [comment]
![11](https://github.com/user-attachments/assets/0dded9db-acab-4c19-9b15-4b115dfda92c)

**Comment by qiuezhe35** [comment]
有卡可抽黑天鹅

**Comment by MoliTecarlly** [comment]
## 我的建议是
抽一下下一个池子的椒丘
现在看来特适合~~雷电将军~~黄泉
具体参考下面视频
[点击跳转B站](https://www.bilibili.com/video/BV1Rm421g7Vo)

**Comment by haaahhhis** [comment]
你这完全可以抽个0+0的云璃
卡妈比较吃环境
你一队补个云璃就完全体了
可以参考下面视频
[这是b站的链接](https://www.bilibili.com/video/BV16S421X7yh)
